sort-table.js

```diff
@@ -0,0 +1,31 @@
+const grid = document.getElementById('grid');
+
+grid.onclick = function(e) {
+  if (e.target.tagName != 'TH') return;
+  sortGrid(e.target.cellIndex, e.target.getAttribute('data-type'));
```

Comment:
fix indentation

destruct properties from an object if you need them more then once